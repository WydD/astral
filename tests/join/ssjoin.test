<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
	<ssjoin>
		<source attributes="id,T,PHYID,temp" id="R_1" sourceAttributes="id,T,PHYID,temp" type="Relation"/>
		<source attributes="temp2,PHYID,T2" id="R_2" sourceAttributes="temp2,PHYID,T2" type="Relation"/>
	</ssjoin>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/join/R_1.R" id="R_1" impl="FileRelationImpl" type="Relation"/>
<source attributes="temp2,PHYID,T2" file="tests/join/R_2.R" id="R_2" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
