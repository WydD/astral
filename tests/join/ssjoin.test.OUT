id,temp,T,temp2,T2
t=1000
1, 20, 1000, 35, 1000
1, 20, 1000, 38, 1000
1, 20, 1000, 35, 1000
1, 20, 1000, 38, 1000
1, 22, 1000, 35, 1000
1, 22, 1000, 38, 1000
t=1050
1, 22, 1000, 32, 1025
1, 18, 1050, 32, 1025
t=1075
1, 22, 1000, 32, 1025
1, 18, 1050, 32, 1025
1, 25, 1075, 32, 1025
